### DISCUSSION TITLE
Does Firebase Dynamic Links work for macOS apps?

### DISCUSSION THREAD

**Discussion by alobaili** [firebase/firebase-ios-sdk#7826] [Q&A] [answered]
I have a native macOS app where I want to support Dynamic Links. I have setup all the required steps following this video: https://www.youtube.com/watch?v=KLBjAg6HvG0

However, clicking on the URL from notes or typing it in a browser does not trigger a call to `application(_:continue:restorationHandler:)` for my app. It just opens the Deep Link URL in a browser.

I'm starting to wonder if Firebase Dynamic Links is supported for macOS apps, or am I doing it wrong.

I can provide more information if necessary. Any help is much appreciated.

**Comment by ncooke3** [comment, marked as answer]
Hi @alobaili, 

Thanks for reaching out and great question! At this time, Dynamic Links is not supported for macOS apps. Check out [this doc](https://firebase.google.com/docs/ios/learn-more) to learn more about which platforms each Firebase product currently supports. If you'd like to see Dynamic Links support for macOS apps, feel free to open a  issue (marked as a *feature request*) and the team will be happy to take a closer look! 🙂

**Reply by paulb777** [reply]
Feature request at #7829

**Comment by benlmyers** [comment]
Hi @alobaili:

I'm a bit late to the party, but if you are looking for a package solution, I have a Swift Package that supports Dynamic Links in macOS using manual URL construction, shortening, and handling. The documentation is still incomplete, but you can find the patch notes [here](https://github.com/Flowductive/easy-firebase/releases/tag/1.4.0) and a link to the package [here](https://github.com/Flowductive/easy-firebase). I hope this helps!
